vms/registration/views.py

```diff
@@ -59,7 +62,20 @@ def post(self, request):
                     request.POST, prefix="admin")
 
                 if user_form.is_valid() and administrator_form.is_valid():
-
+                    password1 = request.POST.get('usr-password')
+                    password2 = request.POST.get('usr-confirm_password')
+                    if not match_password(password1, password2):
+                        self.match_error = True
+                        return render(
+                            request, 'registration/signup_administrator.html',
+                            {
```

Comment:
It would be great if - either, as I mentioned in the other patch - move this context to `def get_context()` - or store it in a variable and use it rather than creating it over and over again.